### PR TITLE
Adds elevation prop to surface components

### PR DESCRIPTION
**Pull request by EverWinter23**
Adds material elevation prop for Surface components for light and dark theme, ranging from 0-24.

### Components with elevation props
- [x] Table
- [x] Transfer
- [x] Calendar
- [x] Collapse, List
- [x] Drawer
- [ ] Side Panel [Can't add-- it's using context]
- [x] Modal
- [x] Popover

Issue https://github.com/sha-el/sha-el-design/issues/11

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -48,17 +52,21 @@ const slideInBottom = keyframes({
...
-    width: '50vw',
-    animation: `${slideInBottom} 0.2s cubic-bezier(0.250, 0.460, 0.450, 0.940) both`,
+const modalStyle = (theme: Theme) => {
+  return stylesheet({
+    container: {
+      maxHeight: '70vh',
+      background: colorShades(theme.background)[1],
+      zIndex: 1001,
+      top: '10vh',
+      boxShadow: shadow('2X', theme),
```
Comment (anitnilay20) [inline src/components/Modal/Modal.tsx:62]: You need to remove these from all the places or else shadows added by elevation will have no effect on it.

Comment (anitnilay20) [reply src/components/Modal/Modal.tsx:62]: Do check it for other components..

**Review by anitnilay20 — CHANGES_REQUESTED** [review]
Looks good. You just need to remove `boxShadow` property which was added earlier.

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (EverWinter23) [reply src/components/Modal/Modal.tsx]: Done.

**Review by anitnilay20 — APPROVED** [review]

**Comment by anitnilay20** [comment]
@EverWinter23 you missed out `boxShadow` in `Card` in component. I have added a fix for it.. looks good to be merged..

**Pull request merged**